docs/2.0/docs/pipelines/guides/stacks.md

```diff
@@ -0,0 +1,44 @@
+# Running Pipelines with Terragrunt Stacks
+
+## Introduction
```

Comment:
It would be nice to include a sentence here conveying the high level benefit of stacks.

Reply:
Nice!